west.yml

```diff
@@ -38,7 +38,7 @@ manifest:
     - name: fw-nrfconnect-zephyr
       path: zephyr
       west-commands: scripts/west-commands.yml
-      revision: 3757f8cfe6ffe2c2b537815bb9ca1ace85882df8
+      revision: pull/119/head
```

Comment:
119 is merged, so this should be updated to 

63a3b13c8d0bb01f273c322e6d9ae665a1ff5c44